CMakeLists.txt

```diff
@@ -134,6 +134,11 @@ option(
     the tools every time you update llvm-project."
     ON
 )
+option(
+    APPLY_LLVM_PERFORMANCE_PATCHES
+    "During checkout, apply optional downstream patches to
+    llvm-project to improve performance."
+)
```

Comment:
This option should probably be described in the comment block on top of the file. 

Should this really be off by default? Users would probably want to get perf improvements by default.

Reply:
The aim is that the default configuration should work out of the box. Downstream changes are additional points of failure (we would like to remove patches we already have), so I would prefer to prioritize the toolchain building first.

Release branches might be a different matter, since they use a defined and therefore stable version of LLVM.